docs/search/query-design/README.md

```diff
@@ -60,16 +60,16 @@ Searches containing more than one token are matched to fields as phrases represe
 Searches containing more than one token are matched to fields as phrases but the degree to which tokens can be reordered is specified. This is a refinement of multi match. 
 
 
-### Stemming
+# Stemming
 
 To control for variation in queries, prefixes and suffixes are removed so that only the root word is matched against fields (eg removal of endings “ly,” “ed” and “ing.”)
 
 
-### Stripping
+# Stripping
 
 To control for variation in punctuation in queries (eg “gray's anatomy,” vs “grays anatomy”), all punctuation is removed before matching.
 
 
-### Shingling
+# Shingling
```

Comment:
We've left these as `h3`s as that's semantically correct. There is the document heading (`h1`), then the section heading (`h2`) and then these as subsections (`h3`) - could we leave them as such?

Reply:
The headings are all supposed to be the same level. Making the bottom ones into h3 makes it look like they are variants of the preceding h1.

Reply:
# Query design (h1) 

## Term (h2)

My bad, shouldn't be `h3`, but definitely `h2`, as should the other terms as `h1` is the title of the document.

Reply:
Keeping them all at the same level makes sense to me, but james definitely has a point about them being `h1`s. Could you add an `h1` title for the whole page and make each section into an `h2` or `h3`?

Reply:
h2 works for me. As long as they are headings of some description and the links in the guide will need to match.